pkg/operator/watchdog_test.go

```diff
@@ -12,13 +12,13 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/utils/pointer"
```

Comment:
Linter was complaining this package was deprecated. The correct replacement seems to be dapr/kit/ptr